AISKU/scripts/listCdnVersions.ps1

```diff
@@ -373,19 +379,28 @@ Function ValidateAccess
 Function GetVersion(
     [string] $name
 ) {
-    $regMatch = '^(.*\/)*([^\/\d]*\.)(\d+(\.\d+)*(-[^\.]+)?)(\.(?:gbl\.js|gbl\.min\.js|cjs\.js|cjs\.min\.js|js|min\.js)(?:\.map)?)$'
+    $regMatch = '^(.*\/)*([^\/\d]*\.)(\d+(\.\d+)*(-[^\.]+)?)(\.(?:gbl\.js|gbl\.min\.js|cjs\.js|cjs\.min\.js|js|min\.js|integrity\.json)(?:\.map)?)$'
```

Comment:
ah so this is to get version from integrity.json file?

Reply:
Yes, this is the list script that we run from the internal DevOps task, this is used to extract and order the output. Examples are in the scripts readme here https://github.com/microsoft/ApplicationInsights-JS/tree/master/AISKU/scripts#listcdnversionsps1